backend/services/restaurants.js

```diff
@@ -24,10 +24,27 @@ const getRestaurantWithFood = async (restaurant) => {
     availability: food.availability,
     price: food.price,
   }));
+  const newFoods = [];
+  await Promise.all(foods.map(async (food) => {
```

Comment:
This can be simplified into newFoods = await Promise.all(...) if the callback in map returns obj